client/src/questions-n-answers/qNa.jsx

```diff
@@ -515,98 +515,105 @@ class QuestionsNAnswers extends React.Component {
-          //  console.log(answerClass)
-
-
+          <div className={this.state.qFormShowOrHide}>
 
+            <UserQuestion
+              recordClick={trackerProps.recordClick}
+              currentItemName={this.props.currentItemName}
+              updateQuestions={this.updateQuestions}
+              qFormShowOrHide={this.state.qFormShowOrHide}
+              addQuestion={this.addQuestion}
+              product_id={this.props.product_id}
+            />
 
-              if (index <= 1) {
-                currentClass = 'questionText'
+          </div>
+          <div className={this.state.aFormShowOrHide}>
+            <UserAnswer
+              recordClick={trackerProps.recordClick}
+              currentItemName={this.props.currentItemName}
+              question_id={this.state.question_id}
+              updateAnswers={this.updateAnswers}
+              addAnswer={this.addAnswer}
+              currentQuestion={this.state.currentQuestion}
+            />
 
-              } else {
-                currentClass = this.state.questionHide
+          </div>
+          <div className={scrollContainerClass? scrollContainerClass : 'questionList container'}>
+            <div className={`List container`}>
+              {this.state.questions.map((question, index) => {
+              let currentClass;
+              let show = false;
+              if (this.state.questionClickCount === 1 && index <= this.state.questionClickCount) {
+                show = true
+              }
+              if (this.state.questionClickCount >= 3 && index <= this.state.questionClickCount) {
+                show = true
               }
-              return <QuestionsContainer
-                      key={index}
-                      addToReported={this.addToReported}
-                      helpfulAnswerClick={this.helpfulAnswerClick}
-                      helpfulQuestionClick={this.helpfulQuestionClick}
-                      currentI={index}
-                      showQuestions={this.showQuestions}
-                      addAnswerScroll={this.addAnswerScroll}
-                      answerTableHide={this.answerTableHide}
-                      answerHide={this.answerHide}
-                      answerHide2={this.answerHide2}
-                      answerClass={answerClass}
-                      lastI={this.state.lastIndex}
-                      answerScroll={this.state.answerScroll}
-                      questionClickCount={this.state.questionClickCount}
-                      answerCount={this.state.answerClickCount}
-                      classname={currentClass}
-                      answers={this.state.answers[index]}
-                      question={question}
-                      addAnswerOnClick={this.addAnswerOnClick}
-                      question_id={question.question_id}
-                      show={show}
-                    />
-            })}
+
+            let answerClass =  this.answerHide2(show)
```

Comment:
maybe get rid of the two in the naming? yeah this comment is nit-picky but just a suggestion